### DISCUSSION TITLE
Overwriting local files with older versions from OneDrive

### DISCUSSION THREAD

**Discussion by dresco** [abraunegg/onedrive#1710] [Q&A] [answered]
Hi, first time running this in a long while, and I've noticed it's trying to overwrite my local files with older versions from OneDrive (I know this isn't supposed to be possible but...)

Not a big problem, as the process was stopped soon enough, but would like to understand how to get a successful sync. Running in verbose dry-run mode, I see the following pattern for each file;

```
The local item has a different modified time 2021-Sep-28 10:20:11Z when compared to remote modified time 2020-Dec-23 14:18:08Z
The local item has a different hash when compared to remote item hash
Local item does not exist in local database - replacing with file from OneDrive - failed download?
Downloading file Data/FreeCAD/Tools/Bit/5mm_Drill.fctb ... done.
```
What would be the correct way to tell it that the local files are the correct ones? Am assuming something like the local-first or upload-only options? Thanks..

**Comment by abraunegg** [comment, marked as answer]
@dresco 
What client version are you using?

The correct option would be to use `--local-first` to get the system in-sync - but also it *should* not try and download the older file if the timestamp is older than the local, so that is a bug.

Please file a bug for this following the support process please.

**Reply by dresco** [reply]
> What client version are you using?

Version is v2.4.13  (fedora 35 package), it must be a good six months since this machine was last synced though, so don't know what version would have been installed when last used.

> Please file a bug for this following the support process please.

Thanks, will do.
